### ISSUE TITLE
Saltbot does not appear to function on Firefox v122.0

### ISSUE THREAD

**Issue by Simmons572** [saltbot-org/saltbot#523]
This issue began a few hours ago today.  The plugin is not visible in the URL bar (cannot figure out how to move the plugin to be grouped with my prexisting pluggins but not really relavent for this).  Additionally, the when opening saltybet.com, the twich chat does not launch in a new tab.  Finally, the console shows no activity, and no automatic betting is occuring.

**Comment by calexil** [comment]
Did you recently update Firefox?

**Comment by Simmons572** [comment]
Yes, it updated automatically after a computer reboot that morning.

**Comment by Simmons572** [comment]
Any updates on this?
